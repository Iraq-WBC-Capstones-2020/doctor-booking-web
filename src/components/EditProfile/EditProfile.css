.editContainer {
  background-color: white;
}

.sideBar {
  background: #ebebeb;
  z-index: 1;
  padding-top: 4rem;
  padding-left: 2rem;
}

.label {
  font-size: 18;
  font-weight: 500;
}

.dayOfweek {
  color: #000;
  font-size: 18px;
  font-weight: 400;
  margin-bottom: 20px;
}

.timeTable p {
  color: #000;
  font-size: 18px;
  font-size: 500;
  padding-left: 5px;
}
/*# sourceMappingURL=EditProfile.css.map */
