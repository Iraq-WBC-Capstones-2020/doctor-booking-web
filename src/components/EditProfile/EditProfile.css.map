{
  "version": 3,
  "mappings": "AAAA,AAAA,cAAc,CAAC;EACb,gBAAgB,EAAE,KAAK;CACxB;;AAED,AAAA,QAAQ,CAAC;EACP,UAAU,EAAE,OAAO;EACnB,OAAO,EAAE,CAAC;EACV,WAAW,EAAE,IAAI;EACjB,YAAY,EAAE,IAAI;CACnB;;AAED,AAAA,MAAM,CAAC;EACL,SAAS,EAAE,EAAE;EACb,WAAW,EAAE,GAAG;CACjB;;AAED,AAAA,UAAU,CAAC;EACT,KAAK,EAAE,IAAI;EACX,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,GAAG;EAChB,aAAa,EAAE,IAAI;CACpB;;AAED,AACE,UADQ,CACR,CAAC,CAAC;EACA,KAAK,EAAE,IAAI;EACX,SAAS,EAAE,IAAI;EACf,SAAS,EAAE,GAAG;EACd,YAAY,EAAE,GAAG;CAClB",
  "sources": [
    "EditProfile.scss"
  ],
  "names": [],
  "file": "EditProfile.css"
}
