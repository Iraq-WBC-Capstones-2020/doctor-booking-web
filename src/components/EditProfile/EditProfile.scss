.editContainer {
  background-color: white;
}

.sidebar {
  background: #f7f7f7;
  z-index: 1;

  .sidbarList {
    padding-top: 100px !important;
    padding-left: 10px !important;
    color: #758095;
  }

  .sidbarList li {
    padding: 0.5rem;
    font-size: 18px;
    font-weight: 400;

    &:hover {
      cursor: pointer;
      color: rgb(119, 119, 214);
    }
  }
}

.label {
  font-size: 18;
  font-weight: 500;
}

.dayOfweek {
  color: #000;
  font-size: 18px;
  font-weight: 400;
  margin-bottom: 20px;
}

.timeTable {
  p {
    color: #000;
    font-size: 18px;
    font-size: 500;
    padding-left: 5px;
  }
}
