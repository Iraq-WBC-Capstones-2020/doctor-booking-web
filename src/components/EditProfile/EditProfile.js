<<<<<<< HEAD
import React from 'react';
import './EditProfile.css';
import avatar from './avatar.svg';
import { Form } from 'react-bootstrap';

function EditProfile() {
  return (
    <div>
      <div className="row p-0 m-0 editContainer">
        <div className="col-md-2 sidebar ">
          <ul className="list-unstyled sidbarList">
            <li>Calendar</li>
            <li>Profile</li>
            <li>Edit profile</li>
          </ul>
        </div>
=======
import React, { useState } from 'react';
import './EditProfile.css';
import Map, { Marker } from 'react-map-gl';
import avatar from './avatar.svg';
import { Form } from 'react-bootstrap';
import SideNav from '../DashboardPage/SideNav/SideNav';

const API =
  'pk.eyJ1IjoiaHVzc2VpbnRhbGFsIiwiYSI6ImNrY3M4dWxwbzFtZDIycnM2OHQ4dXM4cnIifQ.ofCZrIlVF_r4YpQDzSi13g';

function EditProfile() {
  const [viewport, setViewport] = useState({
    latitude: 36.1901,
    longitude: 43.993,
    height: '26rem',
    width: '100%',
    zoom: 10,
  });

  const [marker, setMarker] = useState([0, 0]);
  function placeMarker(e) {
    setMarker(e.lngLat);
    console.log(marker);
  }

  return (
    <div>
      <div className="row p-0 m-0 editContainer">
        <div className="sideBar  col-md-2">
          <SideNav className=" pt-5 mt-5 " />
        </div>

>>>>>>> master
        <div className="col-md-10   px-4 py-4">
          {/* header  */}
          <div className="header d-flex  align-items-center">
            <div className="">
              <img src={avatar} alt="" />
            </div>
            <div className="ml-4">
              <h1>Doctor Name</h1>
              <p>Medical genetics</p>
            </div>
          </div>

          {/* form  */}
          <div className="formInfo  px-4 py-3">
            <form>
              <div className="form-row justify-content-between">
                <div className="col-md-5">
                  <label htmlFor="fullname" className="label">
                    Full name
                  </label>
                  <input
                    type="text"
                    className="form-control"
                    id="fullname"
                    placeholder="enter your fullname"
                    required
                  />
                </div>
                <div className="col-md-5">
                  <label htmlFor="specialty" className="label">
                    Specialty
                  </label>
                  <Form.Control as="select">
                    <option>1</option>
                    <option>2</option>
                    <option>3</option>
                    <option>4</option>
                    <option>5</option>
                  </Form.Control>
                </div>
              </div>
              <div className="form-row my-5 justify-content-between">
                <div className="col-md-5 ">
                  <label htmlFor="experince" className="label">
                    Experince
                  </label>
                  <textarea
                    className="form-control"
                    id="experince"
                    rows="5"
                    placeholder="enter your experince"
                  ></textarea>
                </div>
                <div className="col-md-5 ">
                  <label htmlFor="education" className="label">
                    Education
                  </label>
                  <textarea
                    className="form-control"
                    id="education"
                    rows="5"
                    placeholder="enter your education"
                  ></textarea>
                </div>
              </div>
              <hr />
              <div className="form-row my-5 justify-content-between">
<<<<<<< HEAD
                <div clasName="col-md-5">
=======
                <div className="col-md-5">
>>>>>>> master
                  <label htmlFor="email" className="label">
                    Email
                  </label>
                  <input
                    type="text"
                    className="form-control"
                    id="email"
                    value=""
                    placeholder="enter your email"
                    required
                  />
                </div>
                <div className="col-md-5">
                  <label htmlFor="phoneNumebr" className="label">
                    Phone Number
                  </label>
                  <input
                    type="text"
                    className="form-control"
                    id="phoneNumebr"
                    value=""
                    placeholder="enter your phone number"
                    required
                  />
                </div>
              </div>
              <hr />
              <div className="form-row my-5 justify-content-between">
                <div className="col-md-4  mb-3">
                  <label htmlFor="address" className="label">
                    Address
                  </label>
                  <input
                    type="text"
                    className="form-control"
                    id="address"
                    value="country"
                    required
                  />
                </div>
                <div className="col-md-4 mb-3">
                  <label
                    htmlFor="validationDefault02"
                    className="mt-3 label"
                  ></label>
                  <input
                    type="text"
                    className="form-control"
                    id="validationDefault02"
                    value="city"
                    required
                  />
                </div>
                <div className="col-md-4 mb-3">
                  <label htmlFor="validationDefault02" className="mt-3"></label>
                  <input
                    type="text"
                    className="form-control"
                    id="validationDefault02"
                    value="refrene point"
                    required
                  />
                </div>
              </div>
            </form>
            <div className="row p-3">
<<<<<<< HEAD
              <iframe
                src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d12887.907437953263!2d44.010317449999995!3d36.14277670000001!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40073b4dd69ea69b%3A0xb271c8adc232c062!2sErbil%20Technology%20Institute!5e0!3m2!1sen!2siq!4v1597238804088!5m2!1sen!2siq"
                style={{ width: '100%', border: 0, height: '26rem' }}
                aria-hidden="false"
                tabIndex="0"
                title="location"
              ></iframe>
=======
              <Map
                className="map"
                {...viewport}
                mapboxApiAccessToken={API}
                onViewportChange={(viewport) => setViewport(viewport)}
                onClick={placeMarker}
                mapStyle={'mapbox://styles/mapbox/streets-v11'}
              >
                <Marker longitude={marker[0]} latitude={marker[1]}></Marker>
              </Map>
>>>>>>> master
            </div>

            {/* timetable  */}

            <div className="mt-5 timeTable">
              <p>Time Table</p>
              <div className="row m-2 shadow  rounded-1">
                <div className="col-md-4  p-3 ">
                  <p className="dayOfweek">Sunday</p>
                  <p className="dayOfweek">Monday</p>
                  <p className="dayOfweek">Tuseday</p>
                  <p className="dayOfweek">Wedensday</p>
                  <p className="dayOfweek">Thursday</p>
                  <p className="dayOfweek">Friday</p>
                  <p className="dayOfweek">Sartuday</p>
                </div>
                <div className="col-md-4  p-3 pt-4">
                  <div className="switch-container d-flex mb-4 justify-content-center">
                    <Form.Check
                      type="switch"
                      className="swicth"
                      label=""
                      id="sunday"
                    />
                  </div>
                  <div className="switch-container d-flex mb-4 justify-content-center">
                    <Form.Check
                      type="switch"
                      id="monday"
                      className=""
                      label=""
                    />
                  </div>
                  <div className="switch-container d-flex mb-4 justify-content-center">
                    <Form.Check
                      type="switch"
                      className="swicth"
                      label=""
                      id="tuseday"
                    />
                  </div>
                  <div className="switch-container d-flex mb-4 justify-content-center">
                    <Form.Check
                      type="switch"
                      id="wedensday"
                      className=""
                      label=""
                    />
                  </div>
                  <div className="switch-container d-flex mb-4 justify-content-center">
                    <Form.Check
                      type="switch"
                      className="swicth"
                      label=""
                      id="thursday"
                    />
                  </div>
                  <div className="switch-container d-flex mb-4 justify-content-center">
                    <Form.Check
                      type="switch"
                      id="friday"
                      className=""
                      label=""
                    />
                  </div>
                  <div className="switch-container d-flex mb-4 justify-content-center">
                    <Form.Check
                      type="switch"
                      id="saturday"
                      className=""
                      label=""
                    />
                  </div>
                </div>
                <div className="col-md-4  p-3">
                  <div className="d-flex justify-content-between mb-3">
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm ml-5"
                      placeholder="from"
                    />
                    <span>-</span>
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm mr-5"
                      placeholder="to"
                    />
                  </div>
                  <div className="d-flex justify-content-between mb-3">
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm ml-5"
                      placeholder="from"
                    />
                    <span>-</span>
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm mr-5"
                      placeholder="to"
                    />
                  </div>
                  <div className="d-flex justify-content-between mb-3">
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm ml-5"
                      placeholder="from"
                    />
                    <span>-</span>
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm mr-5"
                      placeholder="to"
                    />
                  </div>
                  <div className="d-flex justify-content-between mb-3">
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm ml-5"
                      placeholder="from"
                    />
                    <span>-</span>
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm mr-5"
                      placeholder="to"
                    />
                  </div>
                  <div className="d-flex justify-content-between mb-3">
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm ml-5"
                      placeholder="from"
                    />
                    <span>-</span>
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm mr-5"
                      placeholder="to"
                    />
                  </div>
                  <div className="d-flex justify-content-between mb-3">
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm ml-5"
                      placeholder="from"
                    />
                    <span>-</span>
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm mr-5"
                      placeholder="to"
                    />
                  </div>
                  <div className="d-flex justify-content-between mb-3">
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm ml-5"
                      placeholder="from"
                    />
                    <span>-</span>
                    <input
                      type="text"
                      className="form-control w-25 form-control-sm mr-5"
                      placeholder="to"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}

export default EditProfile;
