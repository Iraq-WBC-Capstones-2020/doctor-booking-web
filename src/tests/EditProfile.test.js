import React from 'react';
import renderer from 'react-test-renderer';
import EditProfile from '../components/EditProfile/EditProfile';

<<<<<<< HEAD
test('EditProfile page run well run well ', () => {
=======
test('EditProfile page run well ', () => {
>>>>>>> master
  const tree = renderer.create(<EditProfile />).toJSON();
  expect(tree).toMatchSnapshot();
});
