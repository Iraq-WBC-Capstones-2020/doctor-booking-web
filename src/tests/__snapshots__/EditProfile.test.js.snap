// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditProfile page run well  1`] = `
<div>
  <div
    className="row p-0 m-0 editContainer"
  >
    <div
      className="sideBar  col-md-2"
    >
      <div
        className="side-nav-container"
      >
        <div
          className="d-none d-md-block sidebar  nav"
          onKeyDown={[Function]}
        >
          <div
            className="d-flex justify-content-evenly nav-item"
          >
            <img
              alt="profile icon"
              src="person.svg"
            />
            <a
              className="nav-link"
              data-rb-event-key="#!"
              href="#!"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Profile
            </a>
          </div>
          <div
            className="d-flex justify-content-evenly nav-item"
          >
            <img
              alt="edit icon"
              src="edit.svg"
            />
            <a
              className="nav-link"
              data-rb-event-key="#!"
              href="#!"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Edit Profile
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-md-10   px-4 py-4"
    >
      <div
        className="header d-flex  align-items-center"
      >
        <div
          className=""
        >
          <img
            alt=""
            src="avatar.svg"
          />
        </div>
        <div
          className="ml-4"
        >
          <h1>
            Doctor Name
          </h1>
          <p>
            Medical genetics
          </p>
        </div>
      </div>
      <div
        className="formInfo  px-4 py-3"
      >
        <form>
          <div
            className="form-row justify-content-between"
          >
            <div
              className="col-md-5"
            >
              <label
                className="label"
                htmlFor="fullname"
              >
                Full name
              </label>
              <input
                className="form-control"
                id="fullname"
                placeholder="enter your fullname"
                required={true}
                type="text"
              />
            </div>
            <div
              className="col-md-5"
            >
              <label
                className="label"
                htmlFor="specialty"
              >
                Specialty
              </label>
              <select
                className="form-control"
              >
                <option>
                  1
                </option>
                <option>
                  2
                </option>
                <option>
                  3
                </option>
                <option>
                  4
                </option>
                <option>
                  5
                </option>
              </select>
            </div>
          </div>
          <div
            className="form-row my-5 justify-content-between"
          >
            <div
              className="col-md-5 "
            >
              <label
                className="label"
                htmlFor="experince"
              >
                Experince
              </label>
              <textarea
                className="form-control"
                id="experince"
                placeholder="enter your experince"
                rows="5"
              />
            </div>
            <div
              className="col-md-5 "
            >
              <label
                className="label"
                htmlFor="education"
              >
                Education
              </label>
              <textarea
                className="form-control"
                id="education"
                placeholder="enter your education"
                rows="5"
              />
            </div>
          </div>
          <hr />
          <div
            className="form-row my-5 justify-content-between"
          >
            <div
              className="col-md-5"
            >
              <label
                className="label"
                htmlFor="email"
              >
                Email
              </label>
              <input
                className="form-control"
                id="email"
                placeholder="enter your email"
                required={true}
                type="text"
                value=""
              />
            </div>
            <div
              className="col-md-5"
            >
              <label
                className="label"
                htmlFor="phoneNumebr"
              >
                Phone Number
              </label>
              <input
                className="form-control"
                id="phoneNumebr"
                placeholder="enter your phone number"
                required={true}
                type="text"
                value=""
              />
            </div>
          </div>
          <hr />
          <div
            className="form-row my-5 justify-content-between"
          >
            <div
              className="col-md-4  mb-3"
            >
              <label
                className="label"
                htmlFor="address"
              >
                Address
              </label>
              <input
                className="form-control"
                id="address"
                required={true}
                type="text"
                value="country"
              />
            </div>
            <div
              className="col-md-4 mb-3"
            >
              <label
                className="mt-3 label"
                htmlFor="validationDefault02"
              />
              <input
                className="form-control"
                id="validationDefault02"
                required={true}
                type="text"
                value="city"
              />
            </div>
            <div
              className="col-md-4 mb-3"
            >
              <label
                className="mt-3"
                htmlFor="validationDefault02"
              />
              <input
                className="form-control"
                id="validationDefault02"
                required={true}
                type="text"
                value="refrene point"
              />
            </div>
          </div>
        </form>
        <div
          className="row p-3"
        >
          <div
            style={
              Object {
                "cursor": "grab",
                "height": 0,
                "position": "relative",
                "width": 0,
              }
            }
          >
            <div
              style={
                Object {
                  "height": "100%",
                  "position": "relative",
                  "width": "100%",
                }
              }
            >
              <div
                className="map"
                style={
                  Object {
                    "height": "100%",
                    "overflow": "hidden",
                    "position": "absolute",
                    "visibility": "inherit",
                    "width": "100%",
                  }
                }
              />
              <div
                style={
                  Object {
                    "height": 0,
                    "overflow": "visible",
                    "width": 0,
                  }
                }
              />
            </div>
          </div>
        </div>
        <div
          className="mt-5 timeTable"
        >
          <p>
            Time Table
          </p>
          <div
            className="row m-2 shadow  rounded-1"
          >
            <div
              className="col-md-4  p-3 "
            >
              <p
                className="dayOfweek"
              >
                Sunday
              </p>
              <p
                className="dayOfweek"
              >
                Monday
              </p>
              <p
                className="dayOfweek"
              >
                Tuseday
              </p>
              <p
                className="dayOfweek"
              >
                Wedensday
              </p>
              <p
                className="dayOfweek"
              >
                Thursday
              </p>
              <p
                className="dayOfweek"
              >
                Friday
              </p>
              <p
                className="dayOfweek"
              >
                Sartuday
              </p>
            </div>
            <div
              className="col-md-4  p-3 pt-4"
            >
              <div
                className="switch-container d-flex mb-4 justify-content-center"
              >
                <div
                  className="swicth custom-control custom-switch"
                >
                  <input
                    className="custom-control-input"
                    disabled={false}
                    id="sunday"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="sunday"
                    title=""
                  >
                    
                  </label>
                </div>
              </div>
              <div
                className="switch-container d-flex mb-4 justify-content-center"
              >
                <div
                  className="custom-control custom-switch"
                >
                  <input
                    className="custom-control-input"
                    disabled={false}
                    id="monday"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="monday"
                    title=""
                  >
                    
                  </label>
                </div>
              </div>
              <div
                className="switch-container d-flex mb-4 justify-content-center"
              >
                <div
                  className="swicth custom-control custom-switch"
                >
                  <input
                    className="custom-control-input"
                    disabled={false}
                    id="tuseday"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="tuseday"
                    title=""
                  >
                    
                  </label>
                </div>
              </div>
              <div
                className="switch-container d-flex mb-4 justify-content-center"
              >
                <div
                  className="custom-control custom-switch"
                >
                  <input
                    className="custom-control-input"
                    disabled={false}
                    id="wedensday"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="wedensday"
                    title=""
                  >
                    
                  </label>
                </div>
              </div>
              <div
                className="switch-container d-flex mb-4 justify-content-center"
              >
                <div
                  className="swicth custom-control custom-switch"
                >
                  <input
                    className="custom-control-input"
                    disabled={false}
                    id="thursday"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="thursday"
                    title=""
                  >
                    
                  </label>
                </div>
              </div>
              <div
                className="switch-container d-flex mb-4 justify-content-center"
              >
                <div
                  className="custom-control custom-switch"
                >
                  <input
                    className="custom-control-input"
                    disabled={false}
                    id="friday"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="friday"
                    title=""
                  >
                    
                  </label>
                </div>
              </div>
              <div
                className="switch-container d-flex mb-4 justify-content-center"
              >
                <div
                  className="custom-control custom-switch"
                >
                  <input
                    className="custom-control-input"
                    disabled={false}
                    id="saturday"
                    type="checkbox"
                  />
                  <label
                    className="custom-control-label"
                    htmlFor="saturday"
                    title=""
                  >
                    
                  </label>
                </div>
              </div>
            </div>
            <div
              className="col-md-4  p-3"
            >
              <div
                className="d-flex justify-content-between mb-3"
              >
                <input
                  className="form-control w-25 form-control-sm ml-5"
                  placeholder="from"
                  type="text"
                />
                <span>
                  -
                </span>
                <input
                  className="form-control w-25 form-control-sm mr-5"
                  placeholder="to"
                  type="text"
                />
              </div>
              <div
                className="d-flex justify-content-between mb-3"
              >
                <input
                  className="form-control w-25 form-control-sm ml-5"
                  placeholder="from"
                  type="text"
                />
                <span>
                  -
                </span>
                <input
                  className="form-control w-25 form-control-sm mr-5"
                  placeholder="to"
                  type="text"
                />
              </div>
              <div
                className="d-flex justify-content-between mb-3"
              >
                <input
                  className="form-control w-25 form-control-sm ml-5"
                  placeholder="from"
                  type="text"
                />
                <span>
                  -
                </span>
                <input
                  className="form-control w-25 form-control-sm mr-5"
                  placeholder="to"
                  type="text"
                />
              </div>
              <div
                className="d-flex justify-content-between mb-3"
              >
                <input
                  className="form-control w-25 form-control-sm ml-5"
                  placeholder="from"
                  type="text"
                />
                <span>
                  -
                </span>
                <input
                  className="form-control w-25 form-control-sm mr-5"
                  placeholder="to"
                  type="text"
                />
              </div>
              <div
                className="d-flex justify-content-between mb-3"
              >
                <input
                  className="form-control w-25 form-control-sm ml-5"
                  placeholder="from"
                  type="text"
                />
                <span>
                  -
                </span>
                <input
                  className="form-control w-25 form-control-sm mr-5"
                  placeholder="to"
                  type="text"
                />
              </div>
              <div
                className="d-flex justify-content-between mb-3"
              >
                <input
                  className="form-control w-25 form-control-sm ml-5"
                  placeholder="from"
                  type="text"
                />
                <span>
                  -
                </span>
                <input
                  className="form-control w-25 form-control-sm mr-5"
                  placeholder="to"
                  type="text"
                />
              </div>
              <div
                className="d-flex justify-content-between mb-3"
              >
                <input
                  className="form-control w-25 form-control-sm ml-5"
                  placeholder="from"
                  type="text"
                />
                <span>
                  -
                </span>
                <input
                  className="form-control w-25 form-control-sm mr-5"
                  placeholder="to"
                  type="text"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
